readme.md

```diff
@@ -81,11 +80,9 @@ $(document).on("ready", function() {
 })
```

Comment:
also we talked about why not to put the script in the `head`, recommend moving to below closing body, also can remove `.on ready` because it's not needed here